Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,16 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
+
+import Header from "../HeaderComponents/HeaderContainer.js";
+import Content from "./CardContent.js";
+import Footer from "../FooterComponents/Footer.js";
+
```

Comment:
good job with managing your imports and calling them in your render, I know you've been playing with React for awhile so this isn't new but good job anyway lol